netdata.spec.in

```diff
@@ -173,6 +191,7 @@ rm -rf $RPM_BUILD_ROOT
 %if %{with systemd}
 %{_unitdir}/netdata.service
 %else
+%dir %{_sysconfdir}/rc.d/init.d
```

Comment:
Hmm, I will search for additional info on OpenSUSE's use of the `%dir` macro and get back to you on it.